
2022-10-23 10:14:59.021 - [Debug] [Connection] Attacker connected: 78.189.25.224 | Client Identification: SSH-2.0-Go
2022-10-23 10:14:59.048 - [Debug] [Auth] Attacker 112.160.137.62 trying to authenticate with 'password'
2022-10-23 10:14:59.049 - [Debug] [Auto Access] Attacker: 112.160.137.62, Threshold: 1, Attempts: 1
2022-10-23 10:14:59.049 - [Debug] [Auto Access] Compromising the honeypot
2022-10-23 10:14:59.049 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-23 10:14:59.740 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-23 10:14:59.742 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-23 10:14:59.742 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.147
2022-10-23 10:14:59.759 - [Debug] [Auth] Attacker 191.242.188.103 trying to authenticate with 'none'
2022-10-23 10:14:59.760 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 10:14:59.762 - [Debug] [Auth] Attacker 121.200.53.148 trying to authenticate with 'password'
2022-10-23 10:14:59.775 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-23 10:14:59.775 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.147
2022-10-23 10:14:59.890 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:2364956) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_1',
  containerIp: '10.0.3.46',
  mitmPort: 1000,
  autoAccessFixed: 1
}
2022-10-23 10:15:35.105 - [Info] MITM Version: 2
2022-10-23 10:15:35.106 - [Info] Auto Access Enabled: true
2022-10-23 10:15:35.107 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-23 10:15:35.165 - [Info] SSH man-in-the-middle server for 10.0.3.46 listening on 10.0.3.1:1000
2022-10-23 10:15:35.475 - [Debug] [Connection] Attacker connected: 223.171.91.144 | Client Identification: SSH-2.0-Go
2022-10-23 10:15:35.516 - [Debug] [Connection] Attacker connected: 124.221.181.83 | Client Identification: SSH-2.0-Go
2022-10-23 10:15:35.573 - [Debug] [Connection] Attacker connected: 223.171.91.127 | Client Identification: SSH-2.0-Go
2022-10-23 10:15:36.529 - [Debug] [Connection] Attacker connected: 184.67.149.186 | Client Identification: SSH-2.0-Go
2022-10-23 10:15:36.985 - [Debug] [Auth] Attacker 124.221.181.83 trying to authenticate with 'none'
2022-10-23 10:15:36.985 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 10:15:36.993 - [Debug] [Auth] Attacker 223.171.91.144 trying to authenticate with 'none'
2022-10-23 10:15:36.994 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 10:15:37.000 - [Debug] [Auth] Attacker 223.171.91.127 trying to authenticate with 'none'
2022-10-23 10:15:37.000 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 10:15:37.472 - [Debug] [Auth] Attacker 223.171.91.127 trying to authenticate with 'publickey'
2022-10-23 10:15:37.473 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-23 10:15:37.474 - [Debug] [Auth] Attacker: 223.171.91.127 has so far made 1 attempts. Remaining: 5 attempts
2022-10-23 10:15:37.479 - [Debug] [Auth] Attacker 124.221.181.83 trying to authenticate with 'password'
2022-10-23 10:15:37.479 - [Debug] [Auto Access] Attacker: 124.221.181.83, Threshold: 1, Attempts: 1
2022-10-23 10:15:37.479 - [Debug] [Auto Access] Compromising the honeypot
2022-10-23 10:15:37.479 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-23 10:15:37.535 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-23 10:15:37.537 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-23 10:15:37.537 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.46
2022-10-23 10:15:37.548 - [Debug] [Auth] Attacker 223.171.91.144 trying to authenticate with 'password'
2022-10-23 10:15:37.549 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-23 10:15:37.549 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.46
2022-10-23 10:15:37.590 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-23 10:15:37.597 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-23 10:15:38.021 - [Debug] [Connection] Attacker closed the connection
2022-10-23 10:15:38.181 - [Debug] [Auth] Attacker 184.67.149.186 trying to authenticate with 'none'
2022-10-23 10:15:38.181 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-23 10:15:38.203 - [Debug] [Connection] Attacker connected: 147.182.233.56 | Client Identification: SSH-2.0-Go
2022-10-23 10:15:38.253 - [Debug] [Connection] Attacker connected: 223.171.91.127 | Client Identification: SSH-2.0-Go
2022-10-23 10:15:38.398 - [Debug] [Auth] Attacker 184.67.149.186 trying to authenticate with 'publickey'
2022-10-23 10:15:38.398 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-23 10:15:38.399 - [Debug] [Auth] Attacker: 184.67.149.186 has so far made 1 attempts. Remaining: 5 attempts
